src/sentry/incidents/models/alert_rule.py

```diff
@@ -77,6 +78,28 @@ class AlertRuleStatus(Enum):
     DISABLED = 5
 
 
+class AlertRuleDetectionType(models.IntegerChoices):
+    STATIC = 0, gettext_lazy("Static")
+    DYNAMIC = 1, gettext_lazy("Dynamic")
+
+
+class AlertRuleSensitivity(models.IntegerChoices):
+    LOW = 0, gettext_lazy("Low")
+    MEDIUM = 1, gettext_lazy("Medium")
+    HIGH = 2, gettext_lazy("High")
+
+
+class AlertRuleSeasonality(models.IntegerChoices):
```

Comment:
Can we add a note here to call out that we've iterated through all combinations and that we don't foresee expanding this?